frontend/src/components/header/index.js

```diff
@@ -98,14 +98,22 @@ const TabStyled = styled(Tab)(({ theme }) => ({
 const Header = observer(() => {
   const {
     navigationStore: { activeTab, setActiveTab },
-    settingsStore: { robotTypes, robots_settings, currentRobotId, updateCurrentRobotId}
+    settingsStore: { robotTypes, robots_settings, currentRobotId, updateCurrentRobotId },
+    videoPlayerStore: { onShowModal, isStreamStarted },
   } = useStore()
   const theme = useTheme()
-  const [guideOpen, setGuideOpen] = useState(false);
+  const [guideOpen, setGuideOpen] = useState(false)
 
   const handleChange = (_event, newValue) => {
-    setActiveTab(newValue)
-  };
+    const setActiveTabCallback = () => setActiveTab(newValue)
+
+    if(newValue === 1 && isStreamStarted) {
```

Comment:
Is it tab's id? Do we want to map it with names, so that it won't become an issue in case of re-arranging or renaming?

Reply:
done